samcli/commands/check/lib/command_context.py

```diff
@@ -0,0 +1,79 @@
+import os
```

Comment:
Please add documentation about the file/module

Reply:
Added a brief description of the file used in the documentation for "_read_sam_file"